### ISSUE TITLE
User community/mailing list?

### ISSUE THREAD

**Issue by BrenBarn** [emmett-framework/emmett#339]
Back in the Weppy days there was a Google Group, but I don't see anything like that for Emmett.  Is there any kind of mailing list or forum where users can communicate, apart from raising issues directly on the repo?  I'm considering using Emmett for some projects but to be honest one thing making me a bit nervous is I don't have a sense of the size of the userbase and robustness of the project (i.e., how likely it is to die).  The lack of any user community contributes to that uncertainty.

**Comment by gi0baro** [comment]
@BrenBarn the main reason why there's no Google Group for Emmett is the difficulty to track issues/discussion in several different places. Since a few months there's the [Discussions](https://github.com/emmett-framework/emmett/discussions) section in this repository to meet the same need.

Regarding your feelings: surely Emmett doesn't have the users base or community size of other popular projects and honestly I don't have the resources/time to expand the project's market and/or advertise it out there. The only instrument right now is the _word of mouth_ of people liking the project, I had plans for a couple of conferences, but in the last ~1.5 years we had Covid emergency and it made hard to use that channel as well.

Regarding project robustness, I'm the only maintainer since I started in 2014, and of course I cannot have the development speed or power of more popular projects backed by teams of several developers, but for sure in the last 7 years I kept my commitment as much steady as I could, trying to release 3 upgrades per year. Of course I have a real job out there and a life, so it's a matter of having a balance between everything.

I don't have any real numbers to give you, except for the projects driven/contributed by me for companies I worked for, where we had no real issues even with ~5k RPM, and a couple of projects out there I heard of; so again is about my word.
I don't have _the crystal ball_ to tell you about the future or when the project will be in 10 years from now, what I can say is that I have no reasons to stop working on it, so by my side I'll keep going with the same ratio I had since I started.
Of course in case I start getting sponsorship I'll focus more on my OSS projects, but this is a different story.

**Comment by josejachuf** [comment]
Hello @BrenBarn 

I tell you about my experience. Use WEPPY and now Emmett, I think since 2016, on many app, as much as Framework MVC and Rest API and I have never had problems. Most of the time I reported things were fixed immediately. I am very happy with the framework.
I understand what you say, it happens to me who asks me and why you do not use Flask or Django. Besides, I like to develop with Emmett, this works very well and very fast. 

Jose
